README.md

```diff
@@ -92,22 +105,32 @@ To simply run the bot process:
 
 1. Clone this repository.
 
-2. Install Node and `npm`. The instructions [at the NPM website][npm] should suffice.
+2. Install Node v22.14.0 and `npm`. The instructions [at the NPM website][npm] should suffice.
```

Comment:
v22.14 or later?

Reply:
I'm not really a fan of "or later" with Node because my experience has been that later versions can break things and you're better off to stick with the one it was initially built for.  That said, additional minor releases for this stable release are probably fine, but I'm unsure of a quick/concise way to communicate this.